app/elements/element-action-menu/element-action-menu.html

```diff
@@ -47,6 +47,9 @@
     _element: Object,
     iconsOnly: {type: Boolean, value: false, reflectToAttribute: true}
   },
+  repositoryLink: function(source) {
+    return (source.indexOf('http') >= 0) ? source : githubLink(source);
```

Comment:
Hmm... How did that change ever work?  There is no global `githubLink `.  Should that be `this._ githubLink` instead?